src/Bicep.LangServer/Snippets/Templates/res-log-analytics-workspace.bicep

```diff
@@ -0,0 +1,10 @@
+﻿// Log Analytics Workspace
+resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2015-11-01-preview' = {

```

Comment:
The symbolic name here is fixed, but it feels like customers will most likely want to change this. Should we offer a placeholder on the symbolic name?

Reply:
One challenge here is that it will make the authoring of the snippet bicep file harder because we'll get a parse error. However, my understanding was that we were going to have some sort of convention-based replacement. I see that the names of `name` placeholders do match the symbolic names, but I guess the logic to turn symbolic names into placeholders isn't there yet? @bhsubra?

---
In reply to: [611932375](https://github.com/Azure/bicep/pull/2207#discussion_r611932375) [](ancestors = 611932375)

Reply:
@majastrz , do you mean named placeholder for name E.g: ${name: logAnalyticsWorkspace}? We currently don't have that support, but I can try to add some logic to support the same.

Reply:
I'll try to make this change in a follow up PR.